### PR TITLE
Add some tests for the repository assets wrangling options

### LINKED ISSUE #37: Add testing for repository assets wrangling options
Add testing for repository assets wrangling options

### PR DESCRIPTION
**Pull request by gridhead**
Merge this pull request after merging #32, please

Fixes #37

### CODE REVIEW

File: protop2g/work/repo.py [hunk 1]
```diff
@@ -63,86 +60,74 @@ def downdest(self):
...
+        if os.path.exists(os.path.join(self.sloc.name, ".git")):
+            repoobjc = Repo(path=self.sloc.name)
+            brcslist = [
+                refx.name.replace("%s/" % standard.dfremote, "")
+                for refx in repoobjc.remote(standard.dfremote).refs
+            ]
+            standard.sbrcavbl = list(brcslist)
+            return True, brcslist
+        else:
+            return False, "Cloned source namespace assets could not be found"
```
Comment (samyak-jn) [inline protop2g/work/repo.py:77]: I see we have removed the try blocks and expectations area, was there any specific reason for it? The removal of the try-except block means that any exceptions occurring within this code will propagate up to the caller without being handled.

**Review by samyak-jn — COMMENTED** [review]
I just have one query here, otherwise test case and the code seems okay to me!

File: protop2g/work/repo.py (hunk 1 above)
Comment (gridhead) [reply protop2g/work/repo.py:77]: @samyak-jn, yes and the caller is capable of handling the exception.

Adding exception handling in both places, i.e. the caller and the driver makes less sense and makes it difficult to reproduce tests related to them.

Comment (samyak-jn) [reply protop2g/work/repo.py:77]: Coolio, makes sense!

**Review by samyak-jn — APPROVED** [review]

**Pull request merged**